src/pages/_app.tsx

```diff
@@ -9,8 +9,13 @@ import CssBaseline from "@material-ui/core/CssBaseline";
 import theme from "utils/theme";
 import Header from "src/components/Header";
 import Footer from "src/components/Footer";
+import constants from "utils/constants";
+import "../../public/App.css";
+
+export default function MyApp({ Component, pageProps, router }: AppProps) {
+    let admin = false;
+    if (router.pathname.includes("admin")) admin = true;
```

Comment:
would we be able to use the auth from #51 here to check isLoggedIn or sum?

Reply:
I thought about that but I don't see an easy way for it. What we'll probably end up doing is have a fetch call to the `validateLogin` endpoint on every page that needs admin access. The check here is just to see if the admin header should be rendered or not.

Reply:
:+1: